src/elements/element.line.js

```diff
@@ -145,35 +326,106 @@ class Line extends Element {
 		super(props);
 	}
 
-	draw(ctx) {
+	set points(points) {
+		this._children = points;
```

Comment:
In controller.line.js you renamed `_children` to `points`. Does it need to change here to match?

Reply:
I left the internal variable name as it was before, so external code accessing it would not break.
(needed a setter to also invalidate _segments when points change)

Reply:
Do we think `_points` would be more suitable now?